crates/astria-sequencer-relayer/src/types.rs

```diff
@@ -167,6 +169,24 @@ pub struct SequencerBlockData {
     pub rollup_txs: HashMap<Namespace, Vec<IndexedTransaction>>,
 }
 
+impl Hash for SequencerBlockData {
+    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
```

Comment:
If you handroll this (as opposed to deriving it), please provide a test that asserts that symmetry between `PartialEq` and `Hash` hold.